subscription/upgrade.md

```diff
@@ -11,19 +11,19 @@ This page contains information on how to purchase a Docker paid subscription. It
 
 You can upgrade from Docker Personal to a Pro, Team, or a Business subscription from a Personal subscription. When you upgrade to a paid subscription, you can immediately use all the features and entitlements offered in your new subscription. For detailed information on features available in each subscription, see [Docker Pricing](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"}.
 
-## Upgrade from a Personal to a Pro, Team, or Business subscription
+## Upgrade to a Pro, Team, or Business subscription
 
 The following section contains information on how to purchase a Docker  subscription for new customers. To upgrade from a Personal subscription to a Team or Business subscription, you must first convert your personal namespace to an organization namespace. To learn more, see [convert an account into an organization](../docker-hub/convert-account.md).
 
 To purchase a Pro, Team, or Business subscription:
 
 1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account.
 
-2. Click **Upgrade** and select your Pro, Team, or Business subscription.
+2. To upgrade your Personal subscription, click **Upgrade**. To upgrade your Pro or Team subscription, open the drop-down menu next to your username in the top-right corner and select **Billing**. On the **Billing Details** page, click **Change plan** to navigate to the pricing page.
 
    > **Note**
    >
-   > If you have not created an organization for a Team or Business subscription, select **Create Organization** from the drop-down menu and select your plan. Click **Buy Now** and continue with the on-screen instructions.
+   > If you have not created an organization for a Team or Business subscription, select **Create Organization** from the drop-down menu and select your plan. If you already have a subscription, you can also add another organization. Click **Buy Now** and continue with the on-screen instructions.
 
 3. On the **Organization Size** page, select your annual or monthly subscription and update the number of users for your organization. For a Business subscription, the only option available is annually.
```

Comment:
Suggest reword:



>For a Business subscription, the only option available is annually.

Here's another suggestion. Let me know what you think 
 
"If you are upgrading to a Business subscription, you must choose an annual subscription."